### ISSUE TITLE
Questions from the DWYL team regarding Research Resource

### ISSUE THREAD

**Issue by Cleop** [pm5k/tt_research_register#1]
**Questions from the DWYL team regarding Research Resource which any guidance on from tomorrow's meeting would be much appreciated:**

- Please can you give us a breakdown of the user stories by priority?
- The database system will require a unique ID for users, are we right in assuming that the email address which a user uses to sign up to RR can be used for this purpose? or is there another unique ID already in place?
- Once a user has returned their saliva kit their results are manually recorded. Where are these recorded? Will the contents of this information need to be linked at all to their profiles within Redcap or RR aside from noting that the kit has been returned?

#### 3rd Party Project Relationships
At the moment we have the following user journey for signing up to 3rd party projects:
Find a project on RR > read about the project in a summary on RR > click a button on RR which takes you to the 3rd party running the project > sign up to the 3rd party's specific consent/ terms...

This journey is incomplete and we would benefit from knowing the answers to some of the following questions:
- An admin login feature has been mentioned for the creation of new projects (a CMS), at present our understanding is that this login will be solely for RR team members as opposed to 3rd party project leaders. Is this assumption correct?
- The 3rd party will inform RR that a user has been accepted onto a project and the team will log their participation (so this can be displayed on their RR account), how? via the CMS?
- 3rd party projects will update RR on project updates/news manually/verbally which can be viewed on RR by users once updated via the CMS by the RR team?

Some more clarity surrounding the relationship between Qualtrics and Redcap and our access/controls to them... Perhaps @iteles can elaborate on this?

------

@iteles will be adding to this list of questions (most likely this evening) so make sure you check it again tomorrow morning.

**Comment by ellemindwave** [comment]
ANSWERS! Following meeting on Thursday 9th March.

Please can you give us a breakdown of the user stories by priority? **Yes, Elle is currently putting them into Trello and will add to GitHub following kick-off meeting. Elle or Sarah will continue to prioritise issues throughout** 
The database system will require a unique ID for users, are we right in assuming that the email address which a user uses to sign up to RR can be used for this purpose or that we can define a different unique if if needed? Does it have to be numeric for qualtrics or redcap for example? **It doesn't need to be numeric, although this might be beneficial when being used by RedCap. (Something like TTRR0001) Email address can be used for this purpose at the moment as the unique identifier**
Can new fields be added to redcap and if so, what is the process for this? **Yes, it is very easy (Alice can explain all)**
Once a user has returned their saliva kit their results are manually recorded. Will the contents of this information need to be linked at all to their profiles within Redcap or RR aside from noting that the kit has been returned? **No, just that the kit has been returned**
What admin functionality will be required? We're assuming Very basic creation and editing of projects, possible editing of content and allowing admins to mark users as approved on projects - we would suggest that given the timelines, we push for simplicity here (and very little beautifying)
**There would be admins and super-admins. All projects will be added via the TTRR team, so there will be very few admins. 
Admins:
They will be able to add/edit/remove projects
Update the status of projects
Be able to see very basic analytics of number of views / number of consents per project.

Super admins:
They will be able to approve admin status for other users
Change content on the site

No beautifying of this part, that is correct.**

3rd Party Project Relationships

At the moment we have the following user journey for signing up to 3rd party projects:
Find a project on RR > read about the project in a summary on RR > click a button on RR which takes you to the 3rd party running the project > sign up to the 3rd party's specific consent/ terms...

This journey is incomplete and we would benefit from knowing the answers to some of the following questions:

An admin login feature has been mentioned for the creation of new projects (a CMS), at present our understanding is that this login will be solely for RR team members as opposed to 3rd party project leaders. Is this assumption correct? **Yes**

The 3rd party will inform RR that a user has been accepted onto a project and the team will log their participation (so this can be displayed on their RR account) so that the user can be marked as participating in that project. We are assuming this is done in the simplest possible way with an on/off toggle for example next to the user **Yes**

3rd party projects will update RR on project updates/news manually/verbally - we are assuming this will not require changed on RR. Is this the case? **Yes**

Do we have API documentation for Redcap and Qualtrics? **Yes**

Do we have technical points of contact for both of these systems? **Yes**

How are Qualtrics and Redcap linked and what do we have to provide Qualtrics with to ensure that the user info is linked up to the user signing up on RR? Qualtrics feeds into RedCap, and they apparently have a very useful website that should help with knowing what exactly we need to provide to make it work best for this project https://www.qualtrics.com/

**Comment by iteles** [comment]
@ellemindwave Thanks for this!

Please don't add the issues to this github repo yet, I'm keen to start a fresh repository for the new code so we can discuss this tomorrow 👍

**Comment by ellemindwave** [comment]
Yes sure, no worries. I thought you might want to start fresh tomorrow!
